packages/audit/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "jest",
+    "_test": "jest --coverage",
```

Comment:
We can enable it via config for all packages. Like this https://github.com/verdaccio/verdaccio/blob/5.x/jest/config.js#L7 and ignore certain folders that won't be covered.

Reply:
ok

Reply:
I'll push that